services/web/server/src/simcore_service_webserver/projects/projects_handlers_crud.py

```diff
@@ -69,58 +68,89 @@
 routes = web.RouteTableDef()
 
 
-@routes.post(f"/{VTAG}/projects")
+class BaseRequestContext(BaseModel):
+    user_id: UserID = Field(..., alias=RQT_USERID_KEY)
+    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)
+
+
+class ProjectPathParams(BaseModel):
+    project_uuid: UUID = Field(..., alias="project_id")
+
+    class Config:
+        allow_population_by_field_name = True
+
+
+#
+# - Create https://google.aip.dev/133
+#
+
+
+class _ProjectCreateParams(BaseModel):
+    template_uuid: Optional[UUIDStr] = None
+    as_template: Optional[UUIDStr] = None
+    copy_data: bool = True
+    hidden: bool = False
```

Comment:
if we are going the same as the fastapi apps, and for consistency, I think I would prefer having these models in a separate module.

Reply:
IMO for the requests body and response payload I think it makes sense (will do that as soon as I start building "variants" of Project, i.e. ``ProjectGet``, ``ProjectCreate``, etc) . For the parameters I think it is ok having them in place since they are like "arguments" to the handler function and it is easier to manage in place.